### DISCUSSION TITLE
导入自定义插件，显示import问题

### DISCUSSION THREAD

**Discussion by kirass1** [nonebot/discussions#331] [问答]
在nonebot2项目的虚拟环境中显示都有这些包，但是nb run后显示无法导入
![image](https://user-images.githubusercontent.com/78848256/221814231-57c33e92-c1d1-46d4-af58-3033ba3d16bc.png)

**Comment by yanyongyu** [comment]
截图请截完整

**Reply by kirass1** [reply]
![image](https://user-images.githubusercontent.com/78848256/221829030-a71cf982-3417-406b-b779-687122193216.png)
![image](https://user-images.githubusercontent.com/78848256/221829075-6dd1faa7-7206-478d-b161-b802131e6b0e.png)
![image](https://user-images.githubusercontent.com/78848256/221829109-2ca14964-9bb8-45e9-9d17-a47ac08995cb.png)
![image](https://user-images.githubusercontent.com/78848256/221829139-093c3106-0650-49e1-a244-c759af5b0024.png)
![image](https://user-images.githubusercontent.com/78848256/221829387-2c8965f6-d1c3-4aa8-b533-596e1e4d35ea.png)

几个插件都是一样的问题，求教大佬

**Reply by yanyongyu** [reply]
你这不是没有装redis和pandas吗

**Reply by kirass1** [reply]
w问题就是我这个装过了都 而且虚拟环境查看包也都在 然后就是import错误

**Reply by yanyongyu** [reply]
`.venv\Scripts\python.exe -m pip install redis pandas`

**Reply by kirass1** [reply]
解决了，但是又有新问题，就是导入成功了 但输入命令后没有响应
![image](https://user-images.githubusercontent.com/78848256/221858997-6cbdc13b-b112-4c4f-a115-84b784cb8b41.png)
![image](https://user-images.githubusercontent.com/78848256/221859046-2ae353bb-4413-423c-8f38-9d89b27125c8.png)

**Reply by yanyongyu** [reply]
@StarHeartHunt

**Reply by StarHeartHunt** [reply]
这个插件是在哪个平台使用的，能否给一下全对话过程 NoneBot 这边的日志和平台的截图

**Reply by kirass1** [reply]
在飞书使用
![image](https://user-images.githubusercontent.com/78848256/222018165-d9b4b87a-cb6a-4e6d-b257-4473d8673c85.png)
![image](https://user-images.githubusercontent.com/78848256/222018196-fa33a62e-7087-48fd-b7ba-6cf30b931ade.png)
输入/排版机器人没有响应

**Reply by StarHeartHunt** [reply]
是否按[飞书适配器文档](https://feishu.adapters.nonebot.dev/docs/guide/setup#%E9%85%8D%E7%BD%AE%E9%A3%9E%E4%B9%A6%E4%BA%8B%E4%BB%B6%E8%AE%A2%E9%98%85)配置了事件订阅？没有看到事件的日志。

**Reply by kirass1** [reply]
刚刚请求地址错了， 我想要本地运行然后内网穿透，在配置事情请求地址时候没办法返回challenge值
![image](https://user-images.githubusercontent.com/78848256/222025722-b733ff3b-c8d3-4cd2-9cf7-bae38fe76c48.png)
![image](https://user-images.githubusercontent.com/78848256/222025751-9ab14235-d1b4-4e02-917e-47504e4a822c.png)

**Reply by StarHeartHunt** [reply]
入口点错了，按文档格式配置。

**Reply by kirass1** [reply]
这个解决了  谢谢大佬
麻烦问一下docx库使用docx.save()保存文件是不是 与nonebot2有冲突，在本地运行生成一个docx文件打开都是正常的 在nonebot2上跑了后出现内容无法读取的情况
